internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/DaVinciClientTest.java

```diff
@@ -1459,6 +1462,232 @@ public void testIsDavinciHeartbeatReported() throws Exception {
     }
   }
 
+  @Test(timeOut = 2 * TEST_TIMEOUT)
+  public void testBlobP2PTransferAmongDVCWitServerUnexpectedShutdownExpectTriggerIdleEvent() throws Exception {
```

Comment:
I see a lot of duplicate codes between these two new test methods, so can we abstract a common method to reduce the duplication?